test/scenarios-bootc/el10/periodics/el102-src@cis-lvl2.sh

```
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+# CIS hardening runs inside the Robot suite and takes ~20 minutes
+TEST_EXECUTION_TIMEOUT=60m
```

Shellcheck (0.11.0)

[warning] TEST_EXECUTION_TIMEOUT appears unused. Verify use (or export if used externally).

(SC2034)